tests/docker/BUILD

```diff
@@ -390,35 +390,35 @@ file_test(
       }
    ]
 }""",
-    file = "@new_distroless_base//image:index.json",
+    file = "@new_distroless_base//image-oci:index.json",
```

Comment:
Should also test the symlinks in the image directory.

Reply:
added. I also added some for the loader too